charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -105,19 +105,19 @@ Get otel memory_limiter ballast_size_mib value based on 40% of resources.memory.
 Default config override for agent collector deamonset
 */}}
 {{- define "opentelemetry-collector.agentConfigOverride" -}}
+{{- if .Values.standaloneCollector.enabled }}
 exporters:
```

Comment:
`exporters` is a mandatory section for Collector config. Will this result in no `exporters` at all?

Reply:
no, however previous to this change, if standaloneCollector were disabled, `exporters`  would always be `null`

I will provide some example templated manifests before and after this change in the near future